### PR TITLE
git, git-credential-libsecret, git-gui, git-svn 2.37.2

### PR DESCRIPTION
**Pull request by p-linnane**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by carlocab** [comment]
You need to split your single commit into four (one for each formula modified) to enable @BrewTestBot to merge this.

File: Formula/git.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Git < Formula
   desc "Distributed revision control system"
   homepage "https://git-scm.com"
-  url "https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.37.1.tar.xz"
-  sha256 "c8162c6b8b8f1c5db706ab01b4ee29e31061182135dc27c4860224aaec1b3500"
+  url "https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.37.2.tar.xz"
```
Comment (carlocab) [inline Formula/git.rb:4]: Ressource update?

**Comment by carlocab** [comment]
Fixed that for you.

**Comment by p-linnane** [comment]
@carlocab Thank you! I am usually more on the cask side so the linked formulas threw me off.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**